src/Symfony/Component/HttpFoundation/Response.php

```diff
@@ -1210,6 +1210,22 @@ public static function closeOutputBuffers(int $targetLevel, bool $flush): void
         }
     }
 
+    /**
+     * Mark a response as safe according to RFC8674.
+     *
+     * @see https://tools.ietf.org/html/rfc8674
+     */
+    public function setSafeContent(bool $safe = true)
```

Comment:
I'm just wondering: would it be more clear if this were named `setContentSafe`?

Reply:
You're right, I will make the change

Reply:
Done